pact/Pact/Core/ChainData.hs

```diff
@@ -56,6 +57,10 @@ newtype ChainId
   = ChainId { _chainId :: Text }
   deriving (Eq, Show, NFData)
 
+newtype NetworkId
+  = NetworkId { _networkId :: Text }
+  deriving (Eq, Show, NFData)
+
 -- | Allows user to specify execution parameters specific to public-chain
 -- execution, namely gas parameters, TTL, creation time, chain identifier.
 data PublicMeta
```

Comment:
Looks like this is missing JSON instances, this is a blocker for me if so, because it prevents decoding the real `Command PublicMeta ByteString` that we decode in chainweb.